nautobot/extras/tests/test_api.py

```diff
@@ -619,6 +664,174 @@ def test_delete_job_result_with_permission(self):
         self.assertHttpStatus(response, status.HTTP_204_NO_CONTENT)
 
 
+class ScheduledJobTest(
+    APIViewTestCases.GetObjectViewTestCase,
+    APIViewTestCases.ListObjectsViewTestCase,
+):
+    model = ScheduledJob
+    brief_fields = ["interval", "name", "start_time"]
+    choices_fields = []
+
+    @classmethod
+    def setUpTestData(cls):
+        user = User.objects.create(username="user1", is_active=True)
+        ScheduledJob.objects.create(
+            name="test1",
+            task="-",
+            job_class="-",
+            interval=JobExecutionType.TYPE_IMMEDIATELY,
+            user=user,
+            approval_required=True,
+            start_time=now(),
+        )
+        ScheduledJob.objects.create(
+            name="test2",
+            task="-",
+            job_class="-",
+            interval=JobExecutionType.TYPE_IMMEDIATELY,
+            user=user,
+            approval_required=True,
+            start_time=now(),
+        )
+        ScheduledJob.objects.create(
+            name="test3",
+            task="-",
+            job_class="-",
+            interval=JobExecutionType.TYPE_IMMEDIATELY,
+            user=user,
+            approval_required=True,
+            start_time=now(),
+        )
+
+    def test_options_objects_returns_display_and_value(self):
+        """Overriden because this test case is not applicable to this viewset"""
+
+    def test_options_returns_expected_choices(self):
+        """Overriden because this test case is not applicable to this viewset"""
```

Comment:
No luck on figuring out why OPTIONS isn't working? If you want to ping me tomorrow AM I can take another look at it with you.

Reply:
I talked to @jathanism on this and we agreed that this is probably the appropriate solution. The `actions` property in the JSON response for `OPTIONS` gets generated from `metadata_class`, and there is a bit of a weird combination of this being a `ReadOnlyModelViewSet` but it also having `POST` requests defined. We decided that for now, saying that this is out of scope is fine. If any other weird ViewSets like this one emerge, we might need to reconsider.

Reply:
Ok, sounds good. Thanks.